components/image/image.css

```diff
@@ -0,0 +1,16 @@
+@import url("/docs/main.css");
+
+.image_container {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+}
+
+.img_responsive {
+    max-width: 100%;
+    height: auto;
+}
+
+.img_round {
+    border-radius: 50%;
+}
```

Comment:
Hey, just a suggestion if height & width are specified of same size, then it would look like a perfect circle.

Reply:
Like this one:
https://res.cloudinary.com/splashcloud/image/upload/v1644239863/check/check0_vmxtbt.png

Reply:
Thanks Aditya, will make changes!